packages/common/amplify_db_common_dart/lib/src/connect.dart

```diff
@@ -12,6 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'dart:async';
+
+import 'package:drift/backends.dart';
+
 export 'connect_stub.dart'
     if (dart.library.html) 'connect_html.dart'
     if (dart.library.io) 'connect_io.dart';
+
+/// Interface of the [connect].
+typedef Connect = QueryExecutor Function({
```

Comment:
Since the connect function already takes a path parameter do we need to keep the path provider injected dependency as well?

Reply:
Yea I think so, the result of path provide may vary between packages for different use cases.

Reply:
Perhaps the Connect function should already have the path provided?  It appears one needs a path anyway to get a queryExecutor anyway.

Reply:
So it's just exposed with the name parameter.